src/App.js

```diff
@@ -1,9 +1,8 @@
 import React, { Component } from 'react'
-import { injectGlobal } from 'emotion'
+
 import styled from 'react-emotion'
 import activities from './activities'
 import ActivityItem from './components/ActivityItem'
-import logo from './images/discover_compass_search_explorer-512.png'
 
 const Grid = styled('div')`
   display: grid;
```

Comment:
Du hast hier noch keine column Breite etc. definiert... benutzt du das Grid überhaupt irgendwo?

Reply:
ja alexa brauche ich denn weiter unten in Zeile 53 und 66 wird das Grid, was ein gestyltes Div ist verwendet